.github/workflows/nightly.yml

```diff
@@ -5,13 +5,20 @@ on:
     # Every night at midnight
     - cron: "0 0 * * *"
   workflow_dispatch:
+    inputs:
+      rev:
+        description: "Revision hash to run against"
+        required: false
+        default: ""
```

Comment:
Is it to manually select a specific commit to test?

Reply:
Indeed - because if I want to manually test this change, for example, I have to wait until it gets merged.... not great.